src/Approximations/overapproximate.jl

```diff
@@ -1635,3 +1635,66 @@ function overapproximate(X::LazySet{N}, ::Type{<:Zonotope},
     end
     return Zonotope(c, G)
 end
+
+"""
+    overapproximate(r::Rectification{N, <:Zonotope{N}}, ::Type{<:AbstractZonotope}) where {N}
+
```

Comment:
also, the docstring doesn't match the function's signature because of the last  change